build.gradle

```diff
@@ -519,12 +520,32 @@ forbiddenApisTest.enabled = false
  * Maven Publish *
  *****************/
 
+void addManifest(Jar jar) {
+    jar.manifest {
+        attributes 'Implementation-Title': project.name
+        attributes 'Implementation-Vendor': 'HiveMQ'
+        attributes 'Implementation-Version': project.version
+    }
+}
+
+task embeddedJar(type: Jar) {
+    group 'build'
+
+    from sourceSets.main.output
+    addManifest(it)
+    exclude('META-INF/INDEX.LIST', 'META-INF/*.SF', 'META-INF/*.DSA', 'META-INF/*.RSA')
+}
+
+task sourceJar(type: Jar) {
+    from sourceSets.main.allJava
+}
+
 publishing {
     publications {
         distribution(MavenPublication) {
 
             artifact packaging
-
+            artifactId 'hivemq-community-edition'
```

Comment:
not needed, as by default the project name is used

Reply:
ok